Robert Stoelhorst/week_02/day02/youth.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Norbert Wiener Youth</title>
+  <link rel="stylesheet" type="text/css" href="Norbert1.css">
+</head>
+<body>
+  <img src="images/Norbert1.jpeg">
+  <a id="home" href="NorbertW.html">return to home</a>
+
+  <div class="fade"></div>
+
+  <section class="star-wars">
+
+    <div class="crawl">
+
+      <div class="title">
+        <p id="uppercase">mr wiener</p>
```

Comment:
This looks like it should be in a heading tag and not a paragraph tag. This is something that goes a little beyond styling, but is important to have the correct tag structure on your page. If something is a heading it should have a heading tag.